arch/arm/boot/dts/overlays/README

```diff
@@ -3804,6 +3804,21 @@ Params: sel                     The position used for the sel rotary switch.
                                 position. If param is omitted, sel=0 is assumed.
 
 
+Name:   pisound-micro
+Info:   Configures the Blokas Labs Pisound Micro card
+Load:   dtoverlay=pisound-micro
+Params: input-mode             'differential' (default) or 'single-ended'.
```

Comment:
Indents are at fixed character positions. Each of these lines needs an extra space for the body text. Fortunately, there is room.

Reply:
Fixed both README issues.